library/marathon.py

```diff
@@ -231,21 +235,24 @@ def main():
 
     marathon_uri = module.params['uri'].rstrip('/')
     state = module.params['state']
-    json_filename = module.params['app_json']
-    app_json = ''
-    app_id = ''
```

Comment:
can we keep this explicit 'var definition' 👍 
actually even better:
```
appid= None
```
ref: http://stackoverflow.com/questions/664294/is-it-possible-only-to-declare-a-variable-without-assigning-any-value-in-python#664297